python_onnxifi/onnxifi_backend.py

```diff
@@ -0,0 +1,53 @@
+from build.python_onnxifi import *
```

Comment:
`build.python_onnxifi` sounds weird as a package name to me.
Also try not import all from this package. I guess Backend and BackendRep should be enough.